jetson/percep/pcl.cpp

```diff
@@ -378,7 +413,7 @@ void PCL::pcl_obstacle_detection(shared_ptr<pcl::visualization::PCLVisualizer> v
     RANSACSegmentation("remove");
     std::vector<pcl::PointIndices> cluster_indices;
     CPUEuclidianClusterExtraction(cluster_indices);
-    std::vector<std::vector<int>> interest_points(cluster_indices.size(), vector<int> (4));
+    std::vector<std::vector<int>> interest_points(cluster_indices.size(), vector<int> (1000));
```

Comment:
Can you initialize this to 6 and then just push_back into curr_cluster?

Reply:
This seems like it could waste a lot of memory

Reply:
Used push_back